### ISSUE TITLE
Automatic video reproduction blocked on specificated site

### ISSUE THREAD

**Issue by frank67** [sienori/simple-translate#303]
On this site [1] the video reproduction is blocked when simple-translate is active

[1] https://www.rainews.it/tgr/rubriche/piazza-affari/

**Comment by sienori** [comment]
It's not an issue of Simple Translate. Even if I disable the extension, the video may not play.

**Comment by frank67** [comment]
Yes, in Firefox 78.10.0esr (64 bit) disabled all extensions and video won't play. Enabling only Simple Translate the video will play, maybe a FF issue. I apologize for the noise!
